test/javascripts/osm_test.js

```diff
@@ -55,10 +55,10 @@ describe("OSM", function () {
     it("parses bbox params", function () {
       const expected = L.latLngBounds([57.6247, -3.6845], [57.7247, -3.7845]);
       let params = OSM.mapParams("?bbox=-3.6845,57.6247,-3.7845,57.7247");
-      expect(params).to.have.property("bounds").deep.equal(expected);
+      expect(L.latLngBounds(params.bounds)).to.deep.equal(expected);
```

Comment:
The test claims to test `.mapParams`. Why does it need to convert both the expected and the actual value to `L.latLngBounds` before comparing them?